src/vm/sbin/vmadmd.js

```diff
@@ -45,7 +45,9 @@ var qs = require('querystring');
 var SyseventStream = require('/usr/vm/node_modules/sysevent-stream');
 var url = require('url');
 var util = require('util');
+var vasync = require('vasync');
 
+var DOCKER_RUNTIME_DELAY_RESET = 10000; // ms zone being up, we zero delay
```

Comment:
The comment is not very clear without reading the PR's description, maybe writing a longer comment similar to what's described in the PR's description would help.